.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 [type description here, PLEASE, REMOVE THIS LINE, PLACEHOLDER, BEFORE SUBMITTING THIS PULL REQUEST]
 
-- Related ticket: https://progress.opensuse.org/issues/xyz
-- Needles: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/xyz
-- Verification run: openqa.mypersonalinstance.de/tests/xyz#step/module/x
+- Related ticket: [url here or REMOVE line] (eg. https://progress.opensuse.org/issues/xyz)
+- Needles: [url here or REMOVE line] (eg. https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/xyz)
+- Verification run: [url here or REMOVE line] (eg. openqa.mypersonalinstance.de/tests/xyz#step/module/x)
```

Comment:
The format is not very clear. I think what you are trying to do in this template is that every part within square brackets should be removed, right? If this is the case then we would need to state it. This template is getting more and more boiler plate code. What I meant with "best practices" is find good examples in other projects that look nice and tidy and seemingly work.

Reply:
Personal opinion, but I agree with @okurz here - template is too verbose. It's easier / faster / cleaner for me to just (always) remove it and use:
```
Description (sometimes)

Fix for:   #poo or #openqa_job (always)
Local run: #local_job          (most cases)
```